daemon/libnetwork/iptables/firewalld.go

```diff
@@ -366,10 +365,5 @@ type interfaceNotFound struct{ error }
 func (interfaceNotFound) NotFound() {}
 
 func contains(list []string, val string) bool {
-	for _, v := range list {
```

Comment:
For some of these, we should look if we can remove the wrapper / abstraction as well (in a follow-up), as `slices.Contains` is probably equally clear on intent